go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -126,15 +131,62 @@ func (se *Engine) InitDBConfig(cp dbconfigs.Connector) {
 	se.cp = cp
 }
 
+// syncVTDatabase is called either the first time a primary starts or on consequent loads to possibly upgrade to a
+// new Vitess version. This is the only entry point into the sidecardb module to get the _vt database to the desired
+// schema for the running Vitess version.
+// There is some extra logging in here which can be removed in a future version (>v16) once the new schema init
+// functionality is stable.
+func (se *Engine) syncVTDatabase(ctx context.Context, conn *dbconnpool.DBConnection) error {
+	log.Infof("In syncVTDatabase")
+	defer func(start time.Time) {
+		log.Infof("syncVTDatabase took %d ms", time.Since(start).Milliseconds())
+	}(time.Now())
+
+	var exec sidecardb.Exec = func(ctx context.Context, query string, maxRows int, wantFields bool, useVT bool) (*sqltypes.Result, error) {
+		if useVT {
+			_, err := conn.ExecuteFetch(sidecardb.UseVTDatabaseQuery, maxRows, wantFields)
+			if err != nil {
+				return nil, err
+			}
+		}
+		return conn.ExecuteFetch(query, maxRows, wantFields)
+	}
+	log.Infof("before sidecardb.Init")
+	if err := sidecardb.Init(ctx, exec); err != nil {
+		log.Errorf("Error in sidecardb.Init: %+v", err)
+		// temporary logging, to be deleted in v17
+		if strings.Contains(err.Error(), "--read-only") {
+			rs, _ := conn.ExecuteFetch("SHOW GLOBAL VARIABLES LIKE '%read_only%'", 100, false)
+			log.Infof("Readonly variable values: %+v", rs.Rows)
+			log.Infof("Got read-only error from _vt database init %s.\n%s", err.Error(), debug.Stack())
+		}
+		if se.env.Config().DB.HasGlobalSettings() {
+			log.Warning("ignoring sidecardb Init error for unmanaged tablets")
```

Comment:
Minor thing, but we should consistently use sentence capitalization for log messages IMO. More importantly, we should consistently used sidecardb.Init in the related log messages. I say that because logs get noisy and oftentimes you're keyword searching.

Reply:
+1

Reply:
ok